### ISSUE TITLE
How do I generate a list of true and predicted labels after calling ImageClassifier.create() ?

### ISSUE THREAD

**Issue by Tess314** [google/mediapipe#4942]
### Have I written custom code (as opposed to using a stock example script provided in MediaPipe)

None

### OS Platform and Distribution

Windows 10

### Python Version

3.10

### Task name (e.g. Image classification, Gesture recognition etc.)

Image Classification

### Describe the actual behavior

ImageClassifier.create() and model.evaluate() implemented, but need only provides metrics for overall data

### Describe the expected behaviour

Breakdown of true and predicted labels for each element in test dataset

### Standalone code/steps you may have used to try to get what you need

```shell
model = image_classifier.ImageClassifier.create(
    train_data = train_data,
    validation_data = validation_data,
    options=options,
)

loss, acc = model.evaluate(test_data)
print(f'Test loss:{loss}, Test accuracy:{acc}')

import tensorflow as tf

metric_functions = model._metric_functions + [tf.keras.metrics.F1Score(),
 tf.keras.metrics.Precision(),
 tf.keras.metrics.Recall()]

model._model.compile(
    optimizer=model._optimizer,
    loss=model._loss_function,
    metrics=metric_functions,
 )

metrics = model.evaluate(test_data)
print(metrics)
```

**Comment by Tess314** [comment]
Would like output in the format:

| image  | predicted_label  |  true_label |   
|---|---|---|
|  img1 |  dog |  cat |   
|  img2 | cat  |  cat |

**Comment by Tess314** [comment]
Hi there,
Any update on this?

**Comment by Tess314** [comment]
Update?

**Comment by joezoug** [comment]
Hi @Tess314,

Sorry about the delay. You will need to use model.predict to get the predicted_labels. Were you able to resolve the issue in https://github.com/google/mediapipe/issues/4930? That should get you the predicted_labels column, and true_label column will come from the dataset itself.

**Comment by joezoug** [comment]
Hi @Tess314,

Let's continue the discussion in #4930. Please update with what is unclear and I will try to help. Just fyi, our team is quite small so we don't have the bandwidth to write detailed custom code for every user request. We can help answer questions, and have open sourced our codebase so users can look into how our stack is implemented and modify code if needed. Most of our codebase is backed by [Tensorflow](https://www.tensorflow.org/), so that is a good starting point to get familiar with the Model Maker code stack.
